src/plugins/discover/public/application/components/top_nav/get_top_nav_links.tsx

```diff
@@ -241,7 +241,7 @@ export const getTopNavLinks = (
     }),
     run() {
       inspector.open(inspectorAdapters, {
-        title: savedSearch?.title,
+        title: savedSearch?.title || undefined,
```

Comment:
callout: defaulting to undefined just makes sure a title shows up in Inspector panel worst case